### ISSUE TITLE
Post view displays not whole image

### ISSUE THREAD

**Issue by Shura0** [pixelfed/mobile-app#59]
Post view cuts bottom of images and it `s impossible to view whole image

**Comment by RustyPixel** [comment]
I see this when viewing portrait images in dicovery on the app. If website is used the image is displayed correctly

**Comment by DeadSuperHero** [comment]
Yeah, just want to second this. It looks like the timeline maybe forces new pictures to only go up to a certain set height and focuses on the bottom of the picture?

Totally looks okay on web, though.

**Comment by ymau** [comment]
This is an example. I found no way to open a full view. I would have expected that a single tap opened the full view, or that the full view is displayed with "View Post".
![Screenshot_2023-01-02-12-39-27-812_com pixelfed](https://user-images.githubusercontent.com/13555599/210227001-6e314475-5160-4619-a67e-f9fc7df1991a.jpg)
